backend/LearningCalendar/Epicenter.Persistance/Repository/LearningCalendar/EmployeeRepository.cs

```diff
@@ -62,5 +62,14 @@ public async Task<Employee> GetDetailsAsync(string email)
                 .Where(employee => employee.Identity.Email == email)
                 .SingleOrDefaultAsync();
         }
+
+        public async Task<Employee> GetByIdentityId(string identityId)

```

Comment:
Could add `Async` at the end of the name since this is an `async` method and I saw we're doing it